From 476ce22606bc31efaf00eee0f98019407c1a263c Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Wed, 6 Jun 2012 14:29:52 +0100
Subject: [PATCH] configs: Replace CONFIG_PERF_COUNTERS with
 CONFIG_PERF_EVENTS

CONFIG_PERF_COUNTERS was removed in comit 392d65a9 (perf: Remove
PERF_COUNTERS config option)

Signed-off-by: Jon Medhurst <tixy@linaro.org>
---
 linaro/configs/linaro-base.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linaro/configs/linaro-base.conf b/linaro/configs/linaro-base.conf
index c2f23b1..a0fa9d5 100644
--- a/linaro/configs/linaro-base.conf
+++ b/linaro/configs/linaro-base.conf
@@ -7,7 +7,7 @@ CONFIG_IKCONFIG_PROC=y
 CONFIG_LOG_BUF_SHIFT=16
 CONFIG_BLK_DEV_INITRD=y
 CONFIG_EMBEDDED=y
-CONFIG_PERF_COUNTERS=y
+CONFIG_PERF_EVENTS=y
 CONFIG_SLAB=y
 CONFIG_PROFILING=y
 CONFIG_OPROFILE=y
-- 
1.8.0

